Cargo.toml

```diff
@@ -23,9 +23,9 @@ ipld-core = { version = "0.4.2", features = ["serde"] }
 wasmtime = {version = "31.0.0", default-features = false, features = ["cranelift", "pooling-allocator", "parallel-compilation", "runtime"] }
 wasmtime-environ = "31.0.0"
 
-fvm = { path = "fvm", version = "~3.13.0", default-features = false }
-fvm_shared = { path = "shared", version = "~3.13.0", default-features = false }
-fvm_sdk = { path = "sdk", version = "~3.13.0" }
```

Comment:
did we get these (and v2) wrong in the .1 releases?

Reply:
To my understanding, the minor version is completely ignored with the [tilde requirement](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements). That is, `~3.13.0` is equivalent to `~3.13.42`; perhaps we should just use `~3.13` so that the `cargo set-version` doesn't bother with it. What do you think?

Reply:
yeah, I guess it depends on what ends up in the lock, but it does seem like we should just keep them in step